src/app/shared/utils/utils.ts

```diff
@@ -207,6 +207,25 @@ export function getBooleanParamFromTemplateRow(
   return params.hasOwnProperty(name) ? params[name] === "true" : _default;
 }
 
+/** Return a string of space-separated values for the "variant" or "variant_list" parameters  */
+export function getVariantParamFromTemplateRow(
+  row: FlowTypes.TemplateRow,
+  _default: string
+): string {
+  const params = row.parameter_list || {};
+  let variantValue = getParamFromTemplateRow(row, "variant", "");
+  if (params.hasOwnProperty("variant_list")) {
```

Comment:
nit(non-blocking)
Assuming we drop the _list syntax I think this can all be removed